plotly/tools.py

```diff
@@ -2509,6 +2561,205 @@ def create_dendrogram(X, orientation="bottom", labels=None,
+        :param kwargs: kwargs passed through plotly.graph_objs.Heatmap.
+            These kwargs describe other attributes about the annotated Heatmap
+            trace such as the colorscale. For more information on valid kwargs
+            call help(plotly.graph_objs.Heatmap)
+
+        Example 1: Simple annotated heatmap with default configuration
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        z = [[0.300000, 0.00000, 0.65, 0.300000], 
+             [1, 0.100005, 0.45, 0.4300],
+             [0.300000, 0.00000, 0.65, 0.300000],
+             [1, 0.100005, 0.45, 0.00000]]
+
+        hm = FF.create_annotated_heatmap(z)
+        py.iplot(hm)
+        ```
+        """
+        # TODO: protected until #282
+        from plotly.graph_objs import graph_objs
+
+        # Avoiding mutables in the call signature
+        colorscale = colorscale if colorscale is not None else 'RdBu'
+        fontcolor = fontcolor if fontcolor is not None else []
+        FigureFactory._validate_annotated_heatmap(z, annotation_text)
+        annotations = _AnnotatedHeatmap(z, x, y, annotation_text,
+                                        colorscale, fontcolor, reversescale,
+                                        **kwargs).make_annotations()
+
+        if x or y:
+            trace = dict(type='heatmap',
+                         z=z,
+                         x=x,
+                         y=y,
+                         colorscale=colorscale,
+                         showscale=showscale,
+                         **kwargs)
+            layout = dict(annotations=annotations,
+                          xaxis=dict(ticks='', side='top',
+                                     gridcolor='rgb(0, 0, 0)'),
+                          yaxis=dict(ticks='', ticksuffix='  '))
+        else:
+            trace = dict(type='heatmap',
+                         z=z,
+                         colorscale=colorscale,
+                         showscale=showscale,
+                         **kwargs)
+            layout = dict(annotations=annotations,
+                          xaxis=dict(ticks='', side='top',
+                                     gridcolor='rgb(0, 0, 0)',
+                                     showticklabels=False),
+                          yaxis=dict(ticks='', ticksuffix='  ',
+                                     showticklabels=False))
+
+        data = [trace]
+
+        return graph_objs.Figure(data=data, layout=layout)
+
+    @staticmethod
+    def create_table(table_text, colorscale=None, fontcolor=None,
+                     height_constant=30, index=False, index_title='',
+                     hoverinfo='none', **kwargs):
+        """
+        BETA function that creates data tables
+
+        :param (pandas dataframe | list[list]) text: data for table.
+        :param (string|list) colorscale: Colorscale for table where the color
```

Comment:
@theengineear technically they can use a string i.e. `colorscale='Blues'`
you're right `list` should be `list[list]`